### DISCUSSION TITLE
How do I use the command line like the GUI?

### DISCUSSION THREAD

**Discussion by dende82** [enzo1982/freac#502] [Help & Support]
Hello,

I want to do the following: I have multiple folders with FLAC files. I want to convert each folder into MP3 AND a single file. I tried using the command line but it didn't work as intended.

It should be something like this: convert <folder> ; use LAME -v 0 ; single file (name like the default filename when I convert via GUI)

But I couldn't figure out how to

Thanks!

**Comment by Lee-Carre** [comment]
> […] use the command line like the GUI?
> […]
> I tried using the command line but it didn't work as intended.

You could generate & save a configuration profile within the GUI (or edit the configuration file with a text editor, if feeling brave), then tell the CLI to use that profile.

From `freaccmd --help` the following seem relevant:

    Usage:  freaccmd [options] [--] [encoder options] [file(s)]

    […]

      --list-configs                Print a list of available configurations
      --config=<cfg>                Specify configuration to use

> […] convert each folder into MP3 AND a single file.

As for outputting both tracks-as-individual-files **and** the album as a single file, I'm not sure that this is possible in a single pass/operation; you may need to have one configuration profile for each output mode, and run them sequentially. A bit of shell scripting will probably help with this, if you wish it automated or have many FLACs to convert.

> I have multiple folders with FLAC files. I want to convert each folder into MP3 […]

Is there a particular reason for this? Surely FLAC is preferable?

I ask, because of [How To Ask Questions The Smart Way § Before You Ask § Describe the Goal, not the Step](/selfteaching/How-To-Ask-Questions-The-Smart-Way/blob/master/How-To-Ask-Questions-The-Smart-Way.md#describe-the-goal-not-the-step).

**Reply by dende82** [reply]
Hello,
thank you for your reply. I will try it this way.

> Is there a particular reason for this? Surely FLAC is preferable?

Yes, I want to copy some music on my mobile and need to save space, thats why I need to convert some Live albums into mp3 AND single file.

**Reply by Lee-Carre** [reply]
Then Opus would be more efficient, especially from a lossless source.

**Comment by mrfragger** [comment]
Took me a bit of digging to figure out how to run freac from flatpak freac

`flatpak run --command=freaccmd org.freac.freac`

Failed to load module xapp-gtk3-module

```
fre:ac - free audio converter v1.1.7 (x86-64) command line interface
Copyright (C) 2001-2023 Robert Kausch

Usage:	freaccmd [options] [--] [encoder options] [file(s)]

  --encoder=<id>  | -e <id>	Specify the encoder to use (default is LAME)
  --help=<id>     | -h <id>	Print help for encoder specific options

                    -d <dir>	Specify output directory for encoded files
                    -o <file>	Specify output file name in single file mode
  --pattern=<pat> | -p <pat>	Specify output file name pattern (default is "<filename>")

  --list-drives			Print a list of available CD drives

  --drive=<n|id>  | -cd <n|id>	Specify active CD drive (0..n or device path)
  --track=<n>     | -t <n>	Specify input track(s) to rip (e.g. 1-5,7,9 or 'all')
  --timeout=<s>			Timeout for CD track ripping (disabled by default)
  --cddb			Enable CDDB database lookup
  --eject			Eject disc after ripping

  --superfast			Enable SuperFast mode
  --threads=<n>			Specify number of threads to use in SuperFast mode

  --split-chapters		Split to individual chapters if chapter tags are present

  --add-cover=<file>		Add front cover image from file (.png or .jpg)
  --add-cover-back=<file>	Add back cover image from file (.png or .jpg)

  --cuesheet[=<file>]		Create a cue sheet of the conversion results
  --playlist[=<file>]		Create a playlist containing the conversion results

  --ignore-chapters		Do not write chapter information to tags
  --ignore-coverart		Do not write cover images to tags

  --list-configs		Print a list of available configurations
  --config=<cfg>		Specify configuration to use

  --quiet			Do not print any messages

Use -- to separate freaccmd options from encoder options if both have the same name.

Encoder <id> can be one of:

	ffmpeg-alac, fdkaac, flac, lame, meh, mac, mpc, vorbis, opus, sndfile, speex,
	wv, ffmpeg-wma, sndfile-wave
```

**Reply by Lee-Carre** [reply]
Works for me, sans flatpak. So, I'd suspect a flatpak problem.
